SCOPES.md

```diff
@@ -0,0 +1,57 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style beetmover tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to copy to candidates directory in S3 the corresponding release artifacts
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidates-tasks`
+
+* `{scope_prefix}:action:push-to-release`
+  * Tells beetmoverscript to move the artifacts from candidates directory to releases directory for the corresponding release
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-push-tasks`
+
+* `{scope_prefix}:action:push-to-nightly`
+  * Tells beetmoverscript to copy to nightly directory in S3 the corresponding release artifacts
+      * This is **default** if no `action` specified in scopes.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-nightly-tasks`
+  
+* `{scope_prefix}:action:push-to-partner`
+  * Tells beetmoverscript to move the corresponding artifacts of partner releases of Firefox to their corresonding private buckets
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidates-tasks`
+
+* `{scope_prefix}:bucket:release`
+  * Tells beetmoverscript to use the production credentials to access the release buckets (namely candidates and releases)
+  * **Conflicts**: with any other `{scope_prefix}:bucket:*`
+  * **Branch Restrictions**: `all-release-branches`
+
+* `{scope_prefix}:bucket:nightly`
```

Comment:
I've gone with the loose desire to sort alpanumeric on the scopes (which would put `bucket:release` below `bucket:nightly` but I don't feel it is a big deal if we don't)